cg/cli/upload/vogue.py

```diff
@@ -105,6 +108,8 @@ def samples(context, days: int):
 def bioinfo(context, case_name, cleanup, target_load, dry):
     """Load bioinfo case results to the trending database"""
 
+    valid_workflows = ["mip"]
```

Comment:
Suggest to set this as a constant somewhere

Reply:
👍 good idea. Should I keep it in `cg/cli/upload/vogue.py` as well or add it to `cg/constants.py`? I think constants.py is a better place imho.

Reply:
Your call. If it has no value for the rest of (cg) it could be placed at the top of the file

Reply:
Added it top of the file.